tests/wicked/before_test.pm

```diff
@@ -43,6 +45,35 @@ sub run {
     file_content_replace('/etc/sysconfig/network/config', '--sed-modifier' => 'g', '^WICKED_DEBUG=.*' => 'WICKED_DEBUG="all"', '^WICKED_LOG_LEVEL=.*' => 'WICKED_LOG_LEVEL="debug2"');
     #preparing directories for holding config files
     assert_script_run('mkdir -p /data/{static_address,dynamic_address}');
+
+    if (systemctl('is-active NetworkManager', ignore_failure => 1) == 0) {
+        # setup_static_network doesn't work with Network Manager
+        # This configures quickly the interface to be able to install wicked package.
+        # Also, this block switches from NM to Wicked to be able to test Wicked tests.
+
+        # Wicked and NM shouldn't be enabled at the same time.
+        die "wicked and NetworkManager enabled simultaneously " if systemctl('is-active wicked', ignore_failure => 1) == 0;
+
+        my $ip = $self->get_ip(type => 'host', netmask => 1);
+        my $iface = $ctx->iface();
+        assert_script_run('rm -f /etc/NetworkManager/system-connections/*');
+        systemctl("restart NetworkManager");
+        assert_script_run("nmcli connection add type ethernet con-name $iface ifname $iface ip4 $ip gw4 10.0.2.2");
+        configure_static_dns(get_host_resolv_conf());
+        assert_script_run("nmcli con up $iface ifname $iface");
+        record_info('devices', script_output('nmcli device status'));
+        record_info('ip a', script_output('ip a'));
+        record_info('nameserver', script_output('cat /etc/resolv.conf'));
+        assert_script_run('ping -c 5 10.0.2.2');
+        zypper_call("in wicked", timeout => 400);
+        systemctl("enable --force wicked");
+        systemctl("stop NetworkManager");
+        systemctl("disable NetworkManager");
+        power_action('reboot', textmode => 1);
+        $self->wait_boot;
+        $self->select_serial_terminal;
```

Comment:
Any reason, why you use this and not `$self->reboot()` ?

Reply:
It hangs, at least in TW: https://openqa.opensuse.org/tests/2161404#step/before_test/23

Reply:
ok then